tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_persistence.c

```diff
@@ -265,6 +267,10 @@ static void unprovisioned_beacon(uint8_t uuid[16], bt_mesh_prov_oob_info_t oob_i
 {
 	static bool once;
 
+	if (!provisioner_ready) {
```

Comment:
The same here. It'd be well to see the reason why the provisioner cannot start. Any information message is welcome.